test/utils/wabt_engine.cpp

```diff
@@ -8,17 +8,23 @@
 
 #include <test/utils/adler32.hpp>
 #include <test/utils/wasm_engine.hpp>
+#include <algorithm>
 #include <cassert>
 
 namespace fizzy::test
 {
 class WabtEngine final : public WasmEngine
 {
-    wabt::interp::Environment m_env;
-    wabt::interp::DefinedModule* m_module{nullptr};
-
-    // WABT Executor/Thread with default options.
-    wabt::interp::Executor m_executor{&m_env};
+    // mutable because getting RefPtr from Ref requires non-const Store
+    mutable wabt::interp::Store m_store{[] {
+        static wabt::Features features;
```

Comment:
```suggestion
        wabt::Features features;
```
The `static` objects are actually expensive as they must be thread-safe - there might be a mutex used.

Reply:
I changed this, but what mutex?